src/data/queries/vetCenterOutstation.ts

```diff
@@ -0,0 +1,193 @@
+      'field_vet_center_feature_content',
+      PARAGRAPH_RESOURCE_TYPES.FEATURED_CONTENT
+    ),
+    ...getNestedIncludes(
+      'field_health_services',
+      RESOURCE_TYPES.HEALTH_SERVICES
+    ),
+  ])
+}
+
+// Define the option types for the data loader.
+export type VetCenterDataOpts = {
+  id: string
+  context?: ExpandedStaticPropsContext
+}
+
+// Implement the data loader.
+export const data: QueryData<VetCenterDataOpts, NodeVetCenter> = async (
+  opts
+): Promise<NodeVetCenter> => {
+  const entity = (await fetchSingleEntityOrPreview(
+    opts,
+    RESOURCE_TYPES.VET_CENTER,
+    params
+  )) as NodeVetCenter
+  return entity
+}
+
+export const formatter: QueryFormatter<
+  NodeVetCenter,
+  FormattedVetCenterOutstation
+> = (entity: NodeVetCenter) => {
+  // format health services / filter per category
+  const healthServicesArray = queries.formatData(
+    RESOURCE_TYPES.HEALTH_SERVICES,
+    entity.field_health_services
+  )
+  const counselingServicesArray = healthServicesArray.filter(
+    (h) => h && h.vetCenterTypeOfCare === 'counseling'
+  )
+  const referralServicesArray = healthServicesArray.filter(
+    (h) => h && h.vetCenterTypeOfCare === 'referral'
+  )
+  const otherServicesArray = healthServicesArray.filter(
+    (h) =>
+      h &&
+      h.vetCenterTypeOfCare !== 'counseling' &&
+      h.vetCenterTypeOfCare !== 'referral'
+  )
+
+  // format and combine featured content
+  const buildFeaturedContentArray = (
+    ccFeaturedContent,
+    vetCenterFeatureContentArray
+  ) => {
+    if (!ccFeaturedContent || !ccFeaturedContent.fetched) {
+      return vetCenterFeatureContentArray
+    }
+
+    const { field_description, field_section_header, field_cta } =
+      ccFeaturedContent.fetched
+
+    // Centralized content needs to be normalized to match the shape of a standard drupal paragraph in order to be formatted
+    const featuredContentObject = {
+      field_description: field_description[0],
+      field_section_header: field_section_header[0].value,
+      type: PARAGRAPH_RESOURCE_TYPES.FEATURED_CONTENT as FeaturedContent['type'],
+      id: ccFeaturedContent.target_id || '',
+
+      field_cta: {
+        id: field_cta[0].target_id || null,
+        field_button_label: field_cta[0].field_button_label[0].value || null,
+        field_button_link: field_cta[0].field_button_link[0] || null,
+        type: PARAGRAPH_RESOURCE_TYPES.BUTTON as Button['type'],
+        drupal_internal__id: field_cta[0].target_id,
+      },
+    }
+    const combinedArray = [
+      featuredContentObject,
+      ...vetCenterFeatureContentArray,
+    ]
+    const formattedFeaturedContentArray = combinedArray.map((item) =>
+      queries.formatData(PARAGRAPH_RESOURCE_TYPES.FEATURED_CONTENT, item)
+    )
+    return formattedFeaturedContentArray
+  }
+  // Similarly, this formats centralized content FAQs to match what our QA components are expecting
+  const buildFaqs = (faqs) => {
+    const buildQuestionArray = (questions) => {
```

Comment:
Non-blocking recommendation: I like to pull these out into module-level exports so I can write tests for them individually. Looks like you're not relying on any outer-scope variables here, so that lift and shift would be pretty easy.

Reply:
That's a great idea, these overlap with standard VCs so lifting it out would give us something both queries could share instead of re-creating.